templates/tools/dockerfile/run_tests_python_deps_venv.include

```diff
@@ -0,0 +1,29 @@
+#====================
+# run_tests.py python dependencies
+
+# Basic python dependencies to be able to run tools/run_tests python scripts
+# These dependencies are not sufficient to build gRPC Python, gRPC Python
+# deps are defined elsewhere (e.g. python_deps.include)
+RUN apt-get update && apt-get install -y ${'\\'}
+  python3 ${'\\'}
+  python3-pip ${'\\'}
+  python3-setuptools ${'\\'}
+  python3-yaml ${'\\'}
+  python3-venv ${'\\'}
+  && apt-get clean
+
+# Create a python virtual env for installing required dependencies.
+ENV PYTHON_ENV="/opt/venv/grpc-interop"
+RUN python3 -m venv "$PYTHON_ENV"
+RUN echo "source $PYTHON_ENV/bin/activate" >> ~/.bashrc
+# Use bash explicitly as /bin/sh doesn't support the "source" command.
```

Comment:
move this comment to the line right above where `source` is run please?

Reply:
The line above isn't executed immediately, its adding the command to the bashrc. The line below is using bash to activate the virtual environment.